### DISCUSSION TITLE
Geradores estáticos + CMS

### DISCUSSION THREAD

**Discussion by kandebonfim** [frontendbr/forum#227] [Perguntas e Respostas]
Alguém já teve alguma experiência com Geradores estáticos + CMS?

Vi algumas alternativas de CMSs cloud como o [cloudcannon](http://cloudcannon.com/) mas queria um toque da comunidade pra evitar armadilhas.

Tenho familiaridade com o `Middleman` e o `Jekyll` mas aberto a outras soluções também.

**Comment by kandebonfim** [comment]
@willianjusten tem alguma dica?

**Comment by fabricionaweb** [comment]
Eu adoro static generators, estou sempre olhando algum. Mas experiência mesmo tenho pouca. O que você busca exatamente? Velocidade da build? Dê mais detalhes

Você ja deve conhecer, alguns sites que agregam: [StaticGen](https://www.staticgen.com/) e [Static Site Generators](https://staticsitegenerators.net/)

Essa semana o @eminetto postou sua experiência com o [hugo](http://gohugo.io/) (feito em go) 
http://eltonminetto.net/2016/07/11/ola-hugo/

**Comment by willianjusten** [comment]
Pelo que eu entendi @fabricionaweb , a preocupação nem é sobre o gerador e sim se existe um CMS para poder gerenciar e talz...

De fato o Cloudcannon é o mais famoso e completo... De opensource tem o [Marighella](https://github.com/marighella/cms).

Tem o [Siteleaf](http://www.siteleaf.com/) também.

**Comment by eminetto** [comment]
@willianjusten tem um novo serviço sendo lançado: https://forestry.io
Eu não cheguei a testar com detalhes mas parece interessante

**Comment by fabricionaweb** [comment]
Ah agora entendi melhor
Eu conheço o [tinypress](http://tinypress.co), ele usa jekyll mas serve só pra blog

**Comment by kandebonfim** [comment]
@fabricionaweb @willianjusten cheguei a testar o [tinypress](https://tinypress.co/) e o [Siteleaf](http://www.siteleaf.com/) mas eles são feitos especificamente pro jekyll (para criar páginas e posts). Eu queria um mais livre que permitisse a edição dos arquivos que eu escolhesse. Dessa forma eu podia colocar os arquivos de texto pro cliente mudar se achar achar necessário mas não correr o risco dele alterar o que não deve.

Acabei escolhendo usar `middleman/cloudcannon` por enquanto.
